operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md

```diff
@@ -422,38 +489,38 @@ Scenarios where this procedure is applicable:
     }
     ```
 
-    **Discovery troubleshooting**
-
-    The `redfishEndpoint` may cycle between `DiscoveryStarted` and `HTTPsGetFailed` before the endpoint becomes `DiscoverOK`.
-    If the BMC is in `HTTPSGetFailed` for a long period of time, then the following steps may help to determine the cause:
-
-    1. (`ncn-mw#`) Verify that the xname of the BMC resolves in DNS.
-
-        ```bash
-        nslookup x3000c0s38b0
-        ```
-
-        Expected output:
-
-        ```text
-        Server:    10.92.100.225
-        Address:   10.92.100.225#53
-
-        Name:    x3000c0s38b0.hmn
-        Address: 10.254.1.13
-        ```
-
-    1. (`ncn-mw#`) Verify that the BMC is reachable at the expected IP address.
-
-        ```bash
-        ping "${NODE}-mgmt"
-        ```
-
-    1. (`ncn-mw#`) Verify that the BMC Redfish `v1/Managers` endpoint is reachable.
-
-        ```bash
-        curl -k -u root:changeme https://x3000c0s38b0/redfish/v1/Managers
-        ```
+    > ***Discovery troubleshooting***
+    >
+    > The `redfishEndpoint` may cycle between `DiscoveryStarted` and `HTTPsGetFailed` before the endpoint becomes `DiscoverOK`.
+    > If the BMC is in `HTTPSGetFailed` for a long period of time, then the following steps may help to determine the cause:
+    >
+    > 1. (`ncn-mw#`) Verify that the xname of the BMC resolves in DNS.
+    >
+    >    ```bash
+    >    nslookup ${BMC_XNAME}
+    >    ```
+    >
+    >    Expected output:
+    >
+    >    ```text
+    >    Server:    10.92.100.225
+    >    Address:   10.92.100.225#53
+    >
+    >    Name:    x3000c0s38b0.hmn
+    >    Address: 10.254.1.13
+    >    ```
+    >
+    > 1. (`ncn-mw#`) Verify that the BMC is reachable at the expected IP address.
+    >
+    >    ```bash
+    >    ping "${NODE}-mgmt"
+    >    ```
+    >
+    > 1. (`ncn-mw#`) Verify that the BMC Redfish `v1/Managers` endpoint is reachable.
+    >
+    >    ```curl
+    >    curl -f -u root:$IPMI_PASSWORD https://${BMC_XNAME}/redfish/v1/Managers
```

Comment:
Double-checking this works, I can't remember if I needed the `-k` for working around the untrusted https cert.

Think I meant to append `-f` not replace `-k` with it.

Reply:
You need to specify `-k` as we are not currently rolling out certificates generated by the System to BMCs yet.

Reply:
You need to specify -k as we are not currently rolling out certificates generated by the System to BMCs yet.